vector/src/androidTest/java/im/vector/app/core/utils/TemporaryStoreTest.kt

```diff
@@ -19,17 +19,18 @@ package im.vector.app.core.utils
 import org.amshove.kluent.shouldBe
 import org.junit.Test
 import java.lang.Thread.sleep
+import kotlin.time.Duration.Companion.milliseconds
 
 class TemporaryStoreTest {
 
     @Test
     fun testTemporaryStore() {
         // Keep the data 300 millis
-        val store = TemporaryStore<String>(300)
+        val store = TemporaryStore<String>(300.milliseconds)
```

Comment:
Cool syntax! I didn't know there was such helper for durations.

Reply:
This is your TIL, mine was here: https://github.com/vector-im/element-android/pull/6330/files#r899219493 :).

It's new since Kotlin 1.6.